conda_concourse_ci/concourse_config.py

```diff
@@ -458,14 +458,52 @@ def add_convert_task(self, subdir, docker_user=None, docker_pass=None):
                         'mkdir -p converted-artifacts/noarch\n'
                         'find . -name "converted-artifacts" -prune -o -path "*/{subdir}/*.tar.bz2" -print0 | xargs -0 -I file mv file converted-artifacts/{subdir}\n'  # NOQA
                         'find . -name "converted-artifacts" -prune -o -path "*/noarch/*.tar.bz2" -print0 | xargs -0 -I file mv file converted-artifacts/noarch\n'  # NOQA
-                'pushd converted-artifacts/{subdir} && cph t "*.tar.bz2" .conda && popd\n'
-                'pushd converted-artifacts/noarch && cph t "*.tar.bz2" .conda && popd\n'
+                        'pushd converted-artifacts/{subdir} && cph t "*.tar.bz2" .conda && popd\n'  # NOQA
+                        'pushd converted-artifacts/noarch && cph t "*.tar.bz2" .conda && popd\n'  # NOQA
                     .format(subdir=subdir)
                     ],
             }
         }
         self.plan.append({'task': 'convert .tar.bz2 to .conda', 'config': config})
 
+    def add_sign_artifacts_task(self, subdir, signing_key, docker_user=None, docker_pass=None):
+        _source = {
+            'repository': 'public.ecr.aws/y0o4y9o3/anaconda-pkg-build',
+            'tag': 'master-amd64',
+        }
+        if docker_user and docker_pass:
+            _source.update({
+                'username': docker_user,
+                'password': docker_pass
+            })
+        config = {
+            'platform': 'linux',
+            'inputs': [{'name': 'converted-artifacts'}],
+            'outputs': [{'name': 'signed-artifacts'}],
+            'image_resource': {
+                'type': 'docker-image',
+                'source': _source,
+            },
+            'run': {
+                'path': 'sh',
+                'args': [
+                    '-exc',
+                        'mkdir -p signed-artifacts/{subdir}\n'
+                        'mkdir -p signed-artifacts/noarch\n'
+                        'find . -name "signed-artifacts" -prune -o -path "*/{subdir}/*.tar.bz2" -print0 | xargs -0 -I file mv file signed-artifacts/{subdir}\n'  # NOQA
+                        'find . -name "signed-artifacts" -prune -o -path "*/noarch/*.tar.bz2" -print0 | xargs -0 -I file mv file signed-artifacts/noarch\n'  # NOQA
```

Comment:
Do the `.conda` files need to be moved into the signed-artifacts directory as well?

Reply:
I added lines for any `.sig` files, as well.  I'm still a bit lost on why the `add_consolidate_task` is written the way it is.  It seems like it is searching the build directory for files that shouldn't exist, but I'm wondering if it's supposed to be that way to 'consolidate' any artifacts from pre-req build operations.